csharp/Clean Code/ToDo/Program.cs

```
@@ -5,106 +5,88 @@ namespace ToDo
             Console.WriteLine("3. Pending tasks");
             Console.WriteLine("4. Exit");
-
-            // Read line
-            string line = Console.ReadLine();
-            return Convert.ToInt32(line);
+            
+            return Convert.ToInt32(Console.ReadLine());
         }
 
-        public static void ShowMenuDos()
+        public static void AddTask()
         {
-            try
-            {
-                Console.WriteLine("Enter the number of the task to remove: ");
-                // Show current taks
-                for (int i = 0; i < TL.Count; i++)
-                {
-                    Console.WriteLine((i + 1) + ". " + TL[i]);
-                }
-                Console.WriteLine("----------------------------------------");
+            Console.WriteLine("Enter the name of the task: ");
+            string task = Console.ReadLine();
 
-                string line = Console.ReadLine();
-                // Remove one position
-                int indexToRemove = Convert.ToInt32(line) - 1;
-                if (indexToRemove > -1)
-                {
-                    if (TL.Count > 0)
-                    {
-                        string task = TL[indexToRemove];
-                        TL.RemoveAt(indexToRemove);
-                        Console.WriteLine("Task " + task + " deleted");
-                    }
-                }
-            }
-            catch (Exception)
-            {
-            }
+            Tasks.Add(task);
+
+            Console.WriteLine("Task registered successfully");
         }
 
-        public static void ShowMenuAdd()
+        public static void RemoveTask()
         {
-            try
-            {
-                Console.WriteLine("Enter the name of the task: ");
-                string task = Console.ReadLine();
-                TL.Add(task);
-                Console.WriteLine("Task registered successfully");
-            }
-            catch (Exception)
```

Why did you remove the `try-catch` statement?

---

So that I could see the reason of any future failure and fix it instead of the catch block just ignoring it.

---

Ok, got it